File: src/main/java/com/dsm/hwanghae/domains/service/DiaryBookRequestConnectionService.java
package com.dsm.hwanghae.domains.service;

import com.dsm.hwanghae.utils.form.DiaryBookResponseForm;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Headers;

public interface DiaryBookRequestConnectionService {
    @Headers(value = {"accept: application/json", "content-type: application/json"})
    @GET("/repositories/diary-book")
    Call<DiaryBookResponseForm> addRequest(@Header("id") String id);
}

File: src/main/java/com/dsm/hwanghae/domains/service/DiaryBookRequestConnectionService.kt
package com.dsm.hwanghae.domains.service

import com.dsm.hwanghae.utils.form.DiaryBookResponseForm
import retrofit2.Call
import retrofit2.http.*

interface DiaryBookRequestConnectionService {
    @Headers("accept: application/json", "content-type: application/json")
    @GET("/repositories/diary-book")
    fun addRequest(@Header("id") id: String): Call<DiaryBookResponseForm>
}